content/en/blog/2022-03-01-february-update/index.md

```diff
@@ -0,0 +1,361 @@
+- Add support to `kustomize-controller` for making the Kubernetes
+  `Secrets` and `ConfigMaps` referenced in `postBuild.substituteFrom`
+  [optional](/docs/components/kustomize/kustomization/#variable-substitution).
+- Allow dot-prefixed paths to be used for bootstrap e.g. `flux
+  bootstrap --path=".flux/clusters/my-cluster"`.
+- All Flux controllers and libraries are now tested by Google's
+  [continuous fuzzing for open source
+  software](https://github.com/google/oss-fuzz).
+
+### Latest Flagger release is 1.18
+
+[This release](https://github.com/fluxcd/flagger/releases/tag/v1.18.0)
+comes with a new API field called `canaryReadyThreshold` that allows
+setting the percentage of pods that need to be available to consider the
+canary deployment as ready.
+
+Starting with version, the canary `deployment` labels, `annotations` and
+`replicas` fields are copied to the primary deployment at promotion time.
+
+### Security news
+
+#### Security Audit Feedback has been addressed
+
+Back when we announced that the [Flux Security Audit had
+concluded](/blog/2021-11-10-flux-security-audit/),
+we already shared that we had created a [project
+board](https://github.com/orgs/fluxcd/projects/5) where we
+tracked all the concrete and immediately actionable feedback. Weighing
+in at fifty tasks of differing sizes, it took us about 2-3 months
+(depending if you count people's holidays over New Years) to address all
+of them. It was a big team effort across teams, projects and
+disciplines. A big thank you again to everyone who contributed!
+
+Some of these tasks are obviously an ongoing piece of work, for example
+ever expanding the scope of fuzzing, improving our security
+documentation and using our RFC process for defining the future of Flux
+together.
+
+A big push has been made though and Flux is more secure since we all
+came together for this project.
+
+#### Welcome to the Flux Security blog post series
+
+If you read the release notes for the last Flux releases, you will
+notice that a big focus for us has been to make Flux more secure every
+time. As we believe it is equally important to explain in more detail
+what we have done and how you can benefit from these changes, we started
+a blog post series about Flux Security.
+
+[The value of SBOMs](/blog/2022/02/security-the-value-of-sboms/):
+Since the last releases we are also releasing "Software Bill of
+Materials" information, which can be easily parsed programmatically and
+be used in decision making processes about the software in use. Read the
+post to understand how SBOMs are constructed and the different scenarios
+in which they are useful and make you more secure.
+
+[Image Provenance](/blog/2022/02/security-image-provenance/):
+If you have been following news and discussions in the Cloud Native
+space, you will have noticed that this was a hot topic lately. In our
+post we explain why we sign all relevant release artefacts of ours, how
```

Comment:
```suggestion
post we explain why we sign all relevant release artifacts of ours, how
```

I know this is also a British/American English difference, but I think the Flux project is generally spelling it this way.